CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 1.2.0 / 2020-03-11
+
+* [FEATURE] Add an HTTP API to query metadata and runtime information. #184
```

Comment:
Maybe `HTTP API endpoint`?

Reply:
(If you change the date anyway..)

Reply:
It's actually two endpoints. I think talking about an API in general makes more sense.

Reply:
Actually, it should contain "metric" somehow, let me change that.